packages/shared/src/components/modals/SubmitArticleModal.tsx

```diff
@@ -132,33 +133,43 @@ export default function SubmitArticleModal({
     }
   };
 
-  const getButton = (): ReactElement => {
-    if (isSubmitted) {
+  const getAlert = () => {
+    if (existingArticle) {
+      return null;
+    }
+
+    if (!isEnabled) {
       return (
-        <>
-          <div className="flex items-center mt-4 mb-6 h-8 font-bold rounded-10 w-fit bg-theme-status-success text-theme-label-invert padding px-[0.9375rem]">
-            Request sent
-          </div>
-          <p className="mb-2 typo-callout text-theme-label-tertiary">
-            You will be notified via email about the article request status.
-          </p>
-        </>
+        <Alert
+          className="mt-4"
+          type={AlertType.Error}
+          title="You do not have enough reputation to use this feature yet."
+        />
       );
     }
 
     return (
-      <Button
-        className="mt-4 btn-primary "
-        buttonSize="small"
-        type="submit"
-        aria-label={submitArticleModalButton}
-        disabled={!enableSubmission || !isEnabled}
-        loading={isValidating}
-      >
-        <span className={isValidating && 'invisible'}>
-          {submitArticleModalButton}
-        </span>
-      </Button>
+      <Alert className="mt-4" title="Recommendation Guidelines">
+        <AlertParagraph>
+          We want you to submit articles that are well thought out, high value
+          and high quality.
+        </AlertParagraph>
+        <AlertParagraph>
+          Please do not add your own posts, promotional content, clickbait etc.
+          Additionally, redirected or shortened URLs will be rejected, please
+          submit the full and final URL.
+        </AlertParagraph>
+        <AlertParagraph>
+          For more details see our{' '}
+          <a
+            className="hover:underline text-theme-label-link"
+            href="https://docs.daily.dev/docs/how-does-daily-dev-work/community-picks-submission-guidelines"
+            target="_blank"
```

Comment:
Also probably add the rel noopener stuff just in case.

Reply:
Good point. Let me add it.